extensions/readme.md

```diff
@@ -2,7 +2,7 @@
 
 Use subfolders under the `extensions` folder to house extensions to the final unified hosts file.
 
-Currently this repo includes three extensions: 
+Currently this repo includes four extensions: 
 
 * `gambling` for common online betting sites,  
 * `social` for common social media sites, and 
```

Comment:
I think you should also move the "and" as well, since there are four elements, not three.